force-app/main/default/classes/UniversalMocker.cls

```diff
@@ -228,13 +245,24 @@ public with sharing class UniversalMocker implements System.StubProvider {
     }
   }
 
+  private void isCalledForTheNthTime(Integer n) {
+    if (n == null || n < 1) {
```

Comment:
@mitchspano I like the fact that this method uses one-based indexing. However, as a result of my, arguably poor, earlier design choice of using zero-based indexing for `andInvocationNumber` method for retrieving the method parameters, I'd prefer keeping this consistent and use zero-based indexing. I know its confusing, but I'd hate to change the indexing for `andInvocationNumber` at this point and break backwards compatibility :(

Reply:
Ack. I am not certain that the method name `isCalledForTheNthTime` makes sense for the reader if we switched to 0 based indexing. We might need to change the verbiage a little bit.